### ISSUE TITLE
Upgrade CORAL to a modern version of jQuery

### ISSUE THREAD

**Issue by remocrevo** [ndlibersa/resources#37]
We need to upgrade CORAL to use a newer version of jQuery, and we need a plan for staying up to date.

**Comment by PaulPoulain** [comment]
Shouldn't we use the CDN instead of embeding jquery ?
Using 
<script src="//code.jquery.com/jquery-1.11.2.min.js"></script>
gives us a handy way to get the jquery version we want quickly.

We currently use the 1.4.4 it seems, we could have the following plan:
 * use the CDN for 1.4.4 and remove the old jquery
 * update to a newer version (and fix any problem)

Remington, if you agree with this proposal, I can do it (& do a pull request)

**Comment by remocrevo** [comment]
@PaulPoulain yes, you are welcome to work on updating CORAL's version of jQuery. It looks like the calendar/date-picker is using an old custom plugin, so you should probably convert it to use whatever is normal in jQuery now. Thanks for offering!

**Comment by remocrevo** [comment]
And yes, I think the CDN is probably fine.  I can see possible benefits to including the jQuery code with CORAL so that each site hosts it themselves, but I don't know enough about it to make a recommendation.

**Comment by PaulPoulain** [comment]
there's no real pro for hosting jquery yourself (that I know), and there are some cons:
* Using CDN mean that your browser will load jquery once only for all the applications that use (this version) of jquery. Otherwise, each app load one. speed improvement.
 * Using CDN mean you get the fastest access you can imagine, because it's optimized for delivering this content.

**Comment by jeffnm** [comment]
I'm in favor of using a CDN.

**Comment by coraladmin** [comment]
That's three votes for CDN so it might as well proceed.
